test/test_main.py

```diff
@@ -53,6 +53,8 @@ def test_invocation(filename):
 
     if base.startswith("fail-") and not any(description == 'Rule disabled' for description in [result[4] for result in gherkin_results]):
         assert len(results) > 0
+    elif bool(re.search(r'000-activated', base)):
```

Comment:
Alternatively can we just let the files begin with `fail-` `pass-` `activated-`, instead of `pass-por000-activated`. I don't think we expect `fail-...-activated-` cases, or do we?

Reply:
See @civilx64 ? We definitely need that piece of documentation 😂. 
@aothms I agree with the principle, but I think we don't need "activated" unit tests files that much. But rather the "not-activated" ones. More on that from Scott's doc next week

Reply:
Yes, in that case we treat 'activation' similar as 'fail'. There is a behave error triggered, but this is only to indicate that the last then statement has been reached. The only prequisite for this that we limit this 'dummy-rule statement' only for activation(000)-rules. This was the plan anyhow. 

I've created a small script that renames files to start with pass-, or activated- instead of activated after the rule-code. So 
`'pass-aaa000-activated_something.ifc'` to `'activated-aaa000-something.ifc' `and 
`'pass-aaa000-not_activated_something_else` to  `'pass-aaa000-something_else'`
. We don’t expect any fail-...-activated cases, but even if they occur, it won’t be an issue (they'll just be flagged as fail files in the CI/CD). I'll share the script here so we can use it for other activation rules as well (the ones that currently have an open PR). As in the example above, it removes the 'activation/not-activation' part after the rule code in the filename, and if the rule is activated (as indicated by the current text after the rule code), change 'pass' to 'activated'.

```
for file in pass-???000-*; do
  if [[ "$file" == *-activated_* ]]; then
    new_file=$(echo "$file" | sed -e 's/^pass-/activated-/' -e 's/-activated_/-/')
    mv "$file" "$new_file"
  elif [[ "$file" == *-not_activated* ]]; then
    new_file=$(echo "$file" | sed -e 's/-not_activated//')
    mv "$file" "$new_file"
  fi
done
```

> but I think we don't need "activated" unit tests files that much. But rather the "not-activated" one

It allows us to check whether a block will turn grey (not-activated) or green (activated) on the platform without having to manually upload files in UAT. I think it's quite important to include in the CI/CD.

Reply:
This is not what we agreed last Thursday. Activated doesn't correspond to any colour. It's an internal state, users don't care. I would expect test files for not-activated (grey), but why test files for activated? Fail and Pass files are, by definition, activated before they turn one of the two. What am I missing?

Reply:
Activation refers to a state indeed where we can color the card either green or red, indicating that the final step of the validation has been reached and there is a (passing or failing) outcome. However, with the current 000 rule, we don't have any failing files. That's why I mentioned that 'activated' corresponds to green, but this is only true within the context of the current rule
Including these test files in the CI/CD allows us to verify whether a file passes because the validation was successfully executed and completed (reaching the final statement) or if the validation stopped because the activation criteria were not met.

Reply:
Precisely. So for the 000 series, I'd expect only not-activated and pass files. Or not?

Reply:
Currently, we're naming the files 'activated' and 'pass', where 'pass' actually means 'pass because it is not activated'. 
We could reverse it to make this clearer. In that case, with the 'pass' file we do expect a behave error (because of the dummy 'Then' statement; we're not using this statement anywhere else), but then in the CI/CD we'd have to differentiate between 'pass in normative rules' and 'pass with 000-series'. Something like (untested)
```
if base.startswith('pass'): 
    if '000' in base: #should be better pattern matching
          assert len(results) > 0
    elif: 
         assert len(results) == 0
elif base.startswith("fail-") and not any(description == 'Rule disabled' for description in [result[4] for result in     gherkin_results]):
        assert len(results) > 0
elif base.startswith('not-activated')
        assert len(results) == 0
   
```
Note that the scope of these files is purely testing in gherkin, not for end-users. However, I'd agree that we should probably try to keep the used terminology consistent as much as possible because to avoid this kind of confusion.